generators/server/templates/src/main/resources/logback-spring.xml.ejs

```diff
@@ -20,25 +20,23 @@
 <!DOCTYPE configuration>
 
 <configuration scan="true">
-    <!-- Patterns based on https://github.com/spring-projects/spring-boot/blob/v2.7.2/spring-boot-project/spring-boot/src/main/resources/org/springframework/boot/logging/logback/defaults.xml -->
+    <!-- Patterns based on https://github.com/spring-projects/spring-boot/blob/v3.0.0/spring-boot-project/spring-boot/src/main/resources/org/springframework/boot/logging/logback/defaults.xml -->
     <conversionRule conversionWord="crlf" converterClass="<%= packageName %>.config.CRLFLogConverter" />
     <property name="CONSOLE_LOG_PATTERN" value="${CONSOLE_LOG_PATTERN:-%clr(%d{${LOG_DATEFORMAT_PATTERN:-yyyy-MM-dd'T'HH:mm:ss.SSSXXX}}){faint} %clr(${LOG_LEVEL_PATTERN:-%5p}) %clr(${PID:- }){magenta} %clr(---){faint} %clr([%15.15t]){faint} %clr(%-40.40logger{39}){cyan} %clr(:){faint} %crlf(%m){red} %n${LOG_EXCEPTION_CONVERSION_WORD:-%wEx}}"/>
-    <property name="FILE_LOG_PATTERN" value="${FILE_LOG_PATTERN:-%d{${LOG_DATEFORMAT_PATTERN:-yyyy-MM-dd HH:mm:ss.SSS}} ${LOG_LEVEL_PATTERN:-%5p} ${PID:- } --- [%t] %-40.40logger{39} : %crlf(%m) %n${LOG_EXCEPTION_CONVERSION_WORD:-%wEx}}"/>
```

Comment:
@DanielFran This one does use `%crlf(%m)`, which is the `CRLFLogConverter`. Not sure why this is necessary. Its Javadocs don't say what it does. https://github.com/jhipster/generator-jhipster/blob/main/generators/server/templates/src/main/java/package/config/CRLFLogConverter.java.ejs

Reply:
@mraible This is related with https://github.com/jhipster/generator-jhipster/pull/19324. To fix sonar rating it has been introduced this converter.

Reply:
@DanielFran https://github.com/jhipster/generator-jhipster/pull/22407.